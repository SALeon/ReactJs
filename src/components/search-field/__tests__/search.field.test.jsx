import React from 'react';
import { shallow } from 'enzyme';
import SearchField from '../index';

const placeholder = 'placeholder';
const inputHandler = () => ({ });

it('render SearchField component', () => {
  const component = shallow(
    <SearchField
      placeholder={placeholder}
      inputHandler={inputHandler}
    />,
  );

  expect(component).toMatchSnapshot();
});
