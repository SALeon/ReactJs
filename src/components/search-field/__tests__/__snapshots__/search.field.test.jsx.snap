// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render SearchField component 1`] = `
<input
  autoFocus={true}
  className="search-field "
  onChange={[Function]}
  placeholder="placeholder"
  type="text"
/>
`;
